content/privacy.md

```diff
@@ -8,12 +8,19 @@ type: page
 ## Who we are
 
 ## Your personal information
+
 Your personal information belongs to you. Any personal information we get from you will only be used internally and never shared with third parties.
 
+### Contact information
+
+> If we ask for your email address we will only use it to contact you regarding sessions you signed up for, or to notify of future sessions.
```

Comment:
Yes and we could add something about news.

Maybe `notify of future sessions` could become `notify you of future opportunities to engage with Code Reading Clubs`